tests/virus_total_agent_test.py

```diff
@@ -334,3 +335,61 @@ def testVirusTotalAgent_whenFileHasNoPath_shouldReportWithHash(
         "Analysis of the target `44d88612fea8a8f36de82e1278abb02f`:\n|Package|  Result  |  \n"
         "|-------|----------|  \n|Bkav   |_Safe_    |  \n|Elastic|_Malicous_|  \n"
     )
+
+
+def testPrepareTargets_whenIPv4AssetReachCIDRLimit_raiseValueError(
+    mocker: plugin.MockerFixture,
+    scan_message_ipv4_with_mask8: msg.Message,
+    virustotal_agent: virus_total_agent.VirusTotalAgent,
+) -> None:
+    """Test the CIDR Limit in case IPV4 and the Limit is reached."""
+    mocker.patch(
+        "agent.virustotal.scan_url_from_message",
+        return_value={},
+    )
```

Comment:
Do we really need this if `prepare_targets` raises the error before.
Apply to the rest please!
```suggestion
```

Reply:
Absolutely not. since I did these tests using TDD. I forgot to get rid of them.
Thanks.